3.0.0 testing

d4rkstar opened the pull request:
this PR implements the deployer configuration via the package.json file as described in issue [nuvolaris#142](https://github.com/nuvolaris/projects/issues/142)

giusdp committed:
Merge pull request #164 from d4rkstar/3.0.0-php

3.0.0 php

francescotimperioss committed:
chore: using 3.1.0-mastrogpt.2405160951 runtimes build

Commit:
Merge pull request #165 from francescotimperi/3.0.0-testing-ftt

chore: using 3.1.0-mastrogpt.2405160951 runtimes build

d4rkstar committed:
nuvolaris config from package.json

d4rkstar committed:
nuvolaris config from package.json

d4rkstar committed:
Merge branch 'nuvolaris:3.0.0-testing' into 3.0.0-testing

francescotimperioss committed:
chore: adjusted adduser to support activation of object storage (MINIO or CEPH required)

Commit:
Merge pull request #167 from francescotimperi/3.0.0-testing-ftt

upgrade adduser to support generic object storage setup

francescotimperioss committed:
fix: update nuv web upload to use the nuv devel minio upload action to ensure it work on both MINIO and CEPH implementation

Commit:
Merge pull request #168 from francescotimperi/3.0.0-testing-ftt

Ensure nuv web upload works on MINIO and CEPH based deployment

francescotimperioss committed:
fix: auth entry in config.json was overriding AUTH key for newly created wsku resources

Commit:
Merge pull request #169 from francescotimperi/3.0.0-testing-ftt

fix for wsku entries auth key

d4rkstar committed:
Merge branch 'nuvolaris:3.0.0-testing' into 3.0.0-testing

msciabarra committed:
go support

sciabarracom commented:
some notes:
- first rebase as there are a few changes
- fix the deploy 
- refactor so you can use the same list of defaults in all the modules - it can be helpful to create a configuratoin module imported by the other

d4rkstar committed:
Merge remote-tracking branch 'upstream/3.0.0-testing' into 3.0.0-testing

d4rkstar committed:
feat: moved config to module. added pydoc strings

Created a config.py module and moved there all the common defaults params. Added pydoc fucntions documentation

d4rkstar commented:
Hello @sciabarracom i've done all the modifications you requested. Need to better understand with your support what is not working properly in deployer so to fix it.
